src/kirby/components/modal/services/modal.helper.ts

```diff
@@ -4,21 +4,65 @@ import { Animation } from '@ionic/core';
 
 import { ModalConfig } from '../modal-wrapper/config/modal-config';
 import { ModalWrapperComponent } from '../modal-wrapper/modal-wrapper.component';
+import { KirbyAnimation } from '@kirbydesign/designsystem/animation/kirby-animation';
+import { ModalConfigHelper } from '../modal-wrapper/config/modal-config.helper';
 
 @Injectable()
 export class ModalHelper {
   constructor(private ionicModalController: IonicModalController) {}
 
   public async showModalWindow(
-    config: ModalConfig,
+    conf: ModalConfig,
```

Comment:
Keep full property name and rename `const config`to `const mergedConfig` or similar
```suggestion
    config: ModalConfig,
```

Reply:
Fixed